### PR TITLE
Implement email validation

### PR DESCRIPTION
**Pull request by billymohajeri**
The validation  of the contact form - portfolio website
In this milestone, I:

Added validation for email text input.
Added error message for invalid emails.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1273,6 +1273,14 @@ textarea::-webkit-input-placeholder {
...
   }
 
+  #error-label {
+    color: crimson;
+    margin-top: 20px;
+    font-size: 1.2rem;
+    margin-left: 30px;
+    padding: 0;
+  }
+
```
Comment (fherrerao) [inline styles.css:1283]: - [ ] Excellent job adding validation 💯, but there is a minor detail, in the requirements says that the error message should be near the submit button. Kindly, relocate it 🦘

image

**Review by fherrerao — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#
Hi @billymohajeri, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

So Close Snl GIF by Saturday Night Live


## Highlights

✔️  Good use of GitHub flow.
✔️  No linter errors.
✔️  Good README
✔️  Good Commit Messages


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Abenezer-Tilahun) [reply styles.css]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @billymohajeri 

Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Github-Flow.
✔️ Professional README  
✔️ Good Commit Messages


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**